examples/with-next-connect/middleware/auth.js

```diff
@@ -0,0 +1,15 @@
+import cookieSession from 'cookie-session'
+import nextConnect from 'next-connect'
+import passport from '../lib/passport'
+
+const auth = nextConnect()
+  .use(
+    cookieSession({
+      name: 'session',
+      keys: ['hackme'],
+    })
+  )
+  .use(passport.initialize())
```

Comment:
Wait does this mean Passport doesn't have to be used with Express? That's very interesting :thinking:

Reply:
Not at all. It can totally be used with good ol' [connect](https://github.com/senchalabs/connect), except `passport` uses `res.redirect` helper which is only available in `express`.

`next-connect` is pretty much `connect` (hence its name), but with router, sub-application. Yet, it is even 10 times smaller than `connect` in install size.

Reply:
Pretty nice, in that case the `with-passport` example deserves an update, I want to get rid of Express for the smallest thing in the world!, that can work with the auth plugins for passport like google, github, etc, what do you think is the best alternative?

Reply:
I believe our guide in https://nextjs.org/docs/api-routes/api-middlewares will do just fine. Can I give it a try? 😄